upstream-community-operators/seldon-operator/seldonoperator.0.1.2.clusterserviceversion.yaml

```diff
@@ -258,7 +258,7 @@ spec:
   customresourcedefinitions:
     owned:
     - kind: SeldonDeployment
-      name: seldondeployments.machinelearning.seldon.io
+      name: seldondeployments.seldon
```

Comment:
seldondeployments.seldon" is invalid: [spec.group: Invalid value: "seldon": should be a domain with at least one dot.
metadata:name: Use as a name for the CustomResourceDefinition. It must use the {plural-name}.{group} format and use the values from the group and plural fields. If you do not follow these rules, you receive a validation error when installing the CRD.
Read here: https://github.com/kyma-project/community/blob/master/guidelines/custom-resource-definitions.md#location-and-file-name

Reply:
thanks for the heads up @aneeshkp. I just want to confirm that I understand correctly as what I understood from what you had mentioned is that names that look like URLs are invalid, but below you advise to use our url. In that case should we change it back to the previous group - i.e. `seldondeployments.seldon`

Reply:
Oh, I think previously the error was on the version, I thought it was label name too, sorry for the confusion.
You should see it while testing. here is the code that validates the crd's
https://github.com/operator-framework/operator-sdk/blob/master/vendor/k8s.io/apimachinery/pkg/util/validation/validation.go#L74

spec.version, you have 0.1.2, need to fix that.
The version names must contain the word alpha, for example, v1alpha1, if the software contains bugs, if enabling a feature can expose the bugs, and if a feature might be disabled by default.
or 
The version name must contain the word beta, for example, v2beta3, if the software is well-tested, enabling features is safe, has the features enabled by default, and support for the features is available, even though the details can change
or 
A stable definition must be versioned as vX where X is an integer, for example, v1, and it contains features which appear in multiple subsequent versions of the released software.
More information 
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api_changes.md#alpha-beta-and-stable-versions